packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -70,7 +70,7 @@ export const getDocs: () => JSX.Element = () => {
       </Description>
       <Source code={formFactorSnippet} />
       <MessageBar>
-        Note: Only Protrait mode is supported when the `formFactor` is set to "mobile". Landscape mode is not supported.
+        Note: Only Portrait mode is supported when the `formFactor` is set to "mobile". Landscape mode is not supported.
```

Comment:
Thanks for catching this.

I think it's better for mental flow to keep all the package installations in one place. So I'd move this up to the "install the package" section, noting that we will use `uuid` for generating a random groupId later in the quickstart.

That's only one opinion though. Worth asking around in the team.